packages/flutter/lib/src/cupertino/interface_level.dart

```diff
@@ -0,0 +1,66 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import '../widgets/framework.dart';
+
+/// Indicates the visual level for a piece of content.
+enum CupertinoUserInterfaceLevelData {
+  /// The level for your window's main content.
+  base,
+
+  /// The level for content visually above [base].
+  elevated,
+}
+
+/// Establishes a subtree in which [CupertinoUserInterfaceLevel.of] resolve to
```

Comment:
resolve => resolves 

This doesn't really help much. We're going to need to explain what interface level means, why you'd want to change it.